File: sdkwork-app-sdk-csharp/Models/OAuthUrlVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class OAuthUrlVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? AuthUrl { get; set; }
    }
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/OAuthUrlVO.java
package com.sdkwork.app.model;

public class OAuthUrlVO {
    private String createdAt;
    private String updatedAt;
    private String authUrl;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getAuthUrl() {
        return this.authUrl;
    }
    
    public void setAuthUrl(String authUrl) {
        this.authUrl = authUrl;
    }
}
